docs: re-add `Validate AsyncAPI documents` (#2303)
asyncapi/website: pages/docs/guides/validate.md

---
title: "Validate AsyncAPI documents"
description: In this guide, you'll learn multiple ways to validate AsyncAPI documents.
weight: 120
---

## Introduction

> This guide teaches multiple ways to validate AsyncAPI documents. 

Reviewer: ```suggestion
In this guide, you'll learn multiple ways to validate AsyncAPI documents.
```

Official parsers use <a href='https://github.com/asyncapi/spec-json-schemas/'>JSON Schema</a> created for AsyncAPI specification. JSON Schema is not enough to fully validate AsyncAPI documents. Learn more about <a href='https://github.com/asyncapi/spec-json-schemas#custom-validation-needs'>custom JSON Schemas validation needs</a>. Official JavaScript parser supports and validates these special needs.

Take it into account if you're thinking about writing your own parser using official JSON Schema.
</Remember>

### Validation against best practices or company governance rules
Now let's discuss options for validating against best practices or company governance rules, also known as **linting**. When various teams use AsyncAPI, you want to ensure they follow the same rules and are consistent across the organization. It is not enough to validate AsyncAPI documents against official specification rules. 

<Remember>

Let's discuss an example. While the `summary` property is optional in an AsyncAPI document, you could choose to require it for your organization. You would then implement a solution that enables you to enforce internal rules on AsyncAPI documents' providers.

</Remember>

> One way this can be done is by using **Spectral** open source tool. It enables you to define company-specific rules that you can use internally.  

Reviewer: ```suggestion
One way to do this is to use the **Spectral** open-source tool. It enables you to define company-specific rules that you can use internally.  
```
